dev/README_RELEASE_AIRFLOW.md

```diff
@@ -174,6 +182,10 @@ git log apache/v2-2-test --format="%H" -- airflow/sensors/base.py | grep -f /tmp
 
 # Prepare the Apache Airflow Package RC
 
+## Update the milestone
+
+Before cutting an RC, we should look at the milestone and merge anything ready, or if we aren't going to include it in the release we should update the milestone for those issues (and we should do that before cutting the RC - so the milestone gives us an accurate view of what is going to be in the release as soon as we know what it will be)
```

Comment:
```suggestion
Before cutting an RC, we should look at the milestone and merge anything ready, or if we aren't going to include it in the release we should update the milestone for those issues. We should do that before cutting the RC so the milestone gives us an accurate view of what is going to be in the release as soon as we know what it will be.
```

Minor nit, probably better with 2 sentences here.